humpbackdev/humpbackdev.github.io
# Feature/new landing page

To test:

- You need node v6.11.1
- npm install
- gulp watch

Open the index.html created in the root.
Should look like this:

Desktop: 
![_users_beto_development_humpback_index html 1](https://user-images.githubusercontent.com/1081264/35056721-3efacf4e-fb78-11e7-9df0-a759d9f6e193.png)

Tablet:
![_users_beto_development_humpback_index html 3](https://user-images.githubusercontent.com/1081264/35056733-46f9d316-fb78-11e7-8517-4892fa2cc7f2.png)

Mobile:
![_users_beto_development_humpback_index html 2](https://user-images.githubusercontent.com/1081264/35056739-4baf60f6-fb78-11e7-9d1e-1d4e74f05708.png)
